tasks/deploy.ts

```diff
@@ -32,8 +33,15 @@ export const updateConfig = (
     throw new Error('Unknown Chain Id');
   }
 
+  let existingConfig = {};
+  try {
+    existingConfig = parseJsonFile<AddressesConfig>(ADDRESS_FILE);
+  } catch (error) {
+    console.log(`Previous configuration not found at: "${ADDRESS_FILE}"`);
+  }
```

Comment:
```suggestion
  const existingConfig = await Promise.resolve(parseJsonFile<AddressesConfig>(ADDRESS_FILE)).catch(e => {
  console.log(`Previous configuration not found at: "${ADDRESS_FILE}"`);
  })
```
and changing `updateConfig` signature to `async` and calling it in `‎test/tasks/deploy.test.ts‎` (the only found usage) with `await`.

Reply:
Updated to replace the try/catch with a promise